Orange/widgets/gui.py

```diff
@@ -1195,6 +1195,36 @@ def attributeItem(var):
     return attributeIconDict[var], var.name
 
 
+class ListViewWithSizeHint(QListView):
+    def __init__(self, *args, preferred_size=None, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
I suppose I didn't know about the `sizeHint` property. I'll remove the class. Thanks.

No, `QSize` doesn't need to be imported. The class is (was) intended to be used in `gui.listView`, which can accept a tuple and convert it.